dotnet/src/Experimental/Process.Abstractions/KernelProcessStepState.cs

```diff
@@ -31,18 +50,30 @@ public KernelProcessStepState(string name, string? id = null)
         this.Id = id;
         this.Name = name;
     }
+
+    /// <summary>
+    /// Registers a derived type for serialization. Types registered here are used by the KnownType attribute
+    /// to support DataContractSerialization of derived types as required to support Dapr.
+    /// </summary>
+    /// <param name="derivedType">A Type that derives from <typeref name="KernelProcessStepState"/></param>
+    public static void RegisterDerivedType(Type derivedType)
```

Comment:
This is exposed for anyone to invoke?

Reply:
I took another look and figured out how to move all the callers of this to abstractions so this is now internal. Thanks!